src/cli/utils/sharepoint.go

```diff
@@ -42,7 +62,7 @@ func ValidateSharePointRestoreFlags(backupID string, opts SharePointOpts) error
 
 	if _, ok := opts.Populated[FileCreatedAfterFN]; ok && !IsValidTimeFormat(opts.FileCreatedAfter) {
 		fmt.Printf("What was I sent: %v\n", opts.FileCreatedAfter)
-		return errors.New("invalid time format for created-after")
+		return errors.New("invalid time format for " + utils.FileCreatedAfterFN)
```

Comment:
The other flags also need this modification.

Reply:
Resolved with [110b78a](https://github.com/alcionai/corso/pull/2679/commits/110b78af9ba3e0a55eaf8a1b45fd0814502c0f1f)